src/plots/cartesian/layout_attributes.js

```diff
@@ -289,6 +289,21 @@ module.exports = {
             'If *B*, 1B.'
         ].join(' ')
     },
+    exponentbase: {
+        valType: 'number',
+        min: 2,
+        dflt: 10,
+        role: 'style',
+        description: [
+            'Determines the base for axes and exponent labels.',
+            '\"SI\" exponent formatting is available for bases of 2 and 10,',
```

Comment:
:cow2: @mdtusz 

I've tried to be consistent in using `**` (e.g. [here](https://github.com/plotly/plotly.js/blob/master/src/traces/bars/layout_attributes.js#L19)) to denote attribute values in descriptions.

Maybe down the road @chriddyp will use this mini-language in the reference pages.